plugins/pay.c

```diff
@@ -1061,6 +1067,11 @@ static struct command_result *json_pay(struct command *cmd,
 		pc->msat = *msat;
 	}
 
+	if (*fuzzpercent > 0.0) {
```

Comment:
though there's a comment on the old version of fuzzing that mentions that the total amount doesn't exceed the allowed limit for a payment. i wonder if this is still implemented also

Reply:
Hmm yeah I wondered to make it a `do {} while (overpayment > maxfeepercent * amount)`. But the default `fuzzpercent <= maxpercent` and since `fuzzpercent` and `maxpercent` are optional, I think someone providing them knows what they doing ?